### PR TITLE
[Bug 15439] Make possible to set the enabledTracks of a player in LC 7.0

### PR DESCRIPTION
**Pull request by livecodepanos**

### CODE REVIEW

File: engine/src/player-platform.cpp [hunk 1]
```diff
@@ -2520,7 +2520,8 @@ void MCPlayer::getenabledtracks(MCExecPoint &ep)
 }
 #endif
 
-Boolean MCPlayer::setenabledtracks(MCStringRef s)
+//Boolean MCPlayer::setenabledtracks(MCStringRef s)
```
Comment (peter-b) [inline engine/src/player-platform.cpp]: Please delete this line! Is it necessary to keep it commented out?

Comment (livecodepanos) [inline engine/src/player-platform.cpp]: Oops I missed it :)

**Comment by runrevmark** [comment]
@livecode-vulcan 0d1d2ff

**Comment by runrevmark** [comment]
@livecode-vulcan review ok 0d1d2ff

**Comment by livecode-vulcan** [comment]
💙 review by @runrevmark ok 0d1d2ff72a271faa3ca210d814a5c7c0f353dc0f

**Comment by livecode-vulcan** [comment]
😞 test failure 0d1d2ff72a271faa3ca210d814a5c7c0f353dc0f
- try-linux-x86: [failure](https://vulcan.livecode.com/builders/try-linux-x86/builds/190)
- try-win-x86: [failure](https://vulcan.livecode.com/builders/try-win-x86/builds/254)
- try-ios: [failure](https://vulcan.livecode.com/builders/try-ios/builds/143)
- try-android-armv6: [failure](https://vulcan.livecode.com/builders/try-android-armv6/builds/201)
- try-linux-x86_64: [failure](https://vulcan.livecode.com/builders/try-linux-x86_64/builds/136)
- try-mac: unknown

**Pull request merged**